pkg/k8s/k8s.go

```diff
@@ -94,15 +96,27 @@ func NewDeployment(n *v1alpha1.Nginx) (*appv1.Deployment, error) {
 								},
 							},
 							Resources: n.Spec.PodTemplate.Resources,
+							ReadinessProbe: &corev1.Probe{
+								Handler: corev1.Handler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path:   buildHealthcheckPath(n.Spec),
+										Port:   intstr.FromInt(healthcheckPort),
+										Scheme: corev1.URISchemeHTTP,
+									},
+								},
+							},
+						},
+						{
+							Name:  sidecarContainerName,
```

Comment:
I think `nginx-healthchecker` is a better name to this guy.

Reply:
Sure, I will update the container name.